src/CMakeLists.txt

```diff
@@ -97,6 +97,7 @@ message(" ")
 message( STATUS "Configuring Level 5 packages --" )
 add_dir_if_exists( mesh )       # needs c4, ds++, mesh_element
 add_dir_if_exists( quadrature ) # needs mesh_element, parser, special_functions
```

Comment:
This PR needs to be checked with clang-tidy.  There are many simplifications that should be implemented, but it easier to iterate with the tool than for me to list them in this review.  I'll send instructions later.

Reply:
Thanks! Should I do all these edits on the squashed branch or the current branch?